### ISSUE TITLE
[react-devtools-inline][4.22.0]: broken published package

### ISSUE THREAD

**Issue by AviVahl** [facebook/react#22958]
## Steps To Reproduce

1. install `react-devtools-inline` in a project that has `react` and `react-is`
2. open node
3. `require('react-devtools-inline')`
4. See error:
```
Uncaught:
Error: Cannot find module '/Users/jstejada/code/jstejada-react/build/oss-experimental/react-is'
Require stack:
- /home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js
- <repl>
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:933:15)
    at Function.Module._load (node:internal/modules/cjs/loader:778:27)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.14 (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:1344:18)
    at __webpack_require__ (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:21:30)
    at Object.3 (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:5710:17)
    at __webpack_require__ (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:21:30)
    at Object.10 (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:495:64)
    at __webpack_require__ (/home/avi/projects/temp/inlinetest/node_modules/react-devtools-inline/dist/backend.js:21:30) 
```

## The current behavior

Published package contains the following code:

```
module.exports = require("/Users/jstejada/code/jstejada-react/build/oss-experimental/react-is");
...
module.exports = require("/Users/jstejada/code/jstejada-react/build/oss-experimental/react");
```

## The expected behavior

No absolute paths in bundle. successful evaluation.

**Comment by bvaughn** [comment]
Looks like maybe a regression caused by #22813? Can you look into this @jstejada?

**Comment by bvaughn** [comment]
We'l publish a fix for this in 4.22.1

**Comment by bvaughn** [comment]
Fix has been published as 4.22.1. Sorry again.
